infra/Makefile

```diff
@@ -0,0 +1,7 @@
+ENVIRONMENT?=staging
+
+init:
+	cd $(ENVIRONMENT) && terraform init
+
+apply:
```

Comment:
Suggestion: rename to `tf-apply`, since "apply" is a terraform-specific thing

Reply:
https://github.com/bettersg/call-home/pull/87/commits/d082e99765c55115663117c99cc9703b30ad73e2